wiki/Staff_Log/2019/en.md

```diff
@@ -273,34 +273,34 @@ All users listed below were added on 2019-02-16.
 - Moved [MBomb](https://osu.ppy.sh/users/3071175) from Probationary Beatmap Nominators to Beatmap Nominators (2019-11-07)
 - Moved [Rocma](https://osu.ppy.sh/users/566276) from Probationary Beatmap Nominators to Beatmap Nominators (2019-11-07)
 - Added [AirinCat](https://osu.ppy.sh/users/11119539) to Probationary Beatmap Nominators (2019-11-08)
-- Removed [Nao Tomori](https://osu.ppy.sh/users/5364763) from Probationary Beatmap Nominators - resignation (2019-11-08)
+- Removed [Nao Tomori](https://osu.ppy.sh/users/5364763) from Probationary Beatmap Nominators—resignation (2019-11-08)
```

Comment:
all of this stuff in the staff log should be left as is. em dash doesn't work here